source/query-with-natural-language/prompt-natural-language-query.txt

```diff
@@ -9,6 +9,100 @@ Prompt your First Natural Language Query
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 1
 
-Prompt your First Natural Language Query
\ No newline at end of file
+You can use MongoDB Compass to generate filter documents using the
+natural language querying feature. This feature automatically generates 
+filter documents based on prompts you provide to the MongoDB AI engine. 
+Natural language querying serves as a helpful starting point and assists 
+you in learning to write MongoDB queries.
+
+About this Task
+---------------
+
+.. note::
+
+   .. include:: /includes/fact-natural-language-query.rst
+
+- The MongoDB AI engine can return both filter documents and 
+  aggregations. If your prompt results in an aggregation, a pop-up menu
+  directs you to the :guilabel:`Aggregations` tab.
```

Comment:
can we also callout that users can use natural language directly on the aggregations tab and link that page? reading this in isolation, it makes it seem like the preferred workflow is to type all types of prompts into the filter experience, when really this behavior is more error handling